maro/simulator/scenarios/data_center/virtual_machine.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
```

Comment:
so we need a VM config, which only includes the cpu/memory requirement, add a property: size_config?

Reply:
requirement_size, deployment_size

Reply:
or required_resource.cpu/required_resource.memory
deploied_resource.cpu/deploied_resource.memroy